chainweb.cabal

```diff
@@ -509,7 +509,7 @@ test-suite chainweb-tests
 
     build-depends:
         -- internal
-          chainweb
+        , chainweb
```

Comment:
oof

Reply:
i prefer leading commas for easy re-arranging (eg with vim's `:sort u`), but i can undo this. originally it was because i added a temporary dependency above this